wagtail/admin/views/account.py

```diff
@@ -138,10 +138,18 @@ def render(self):
 
 class NameEmailSettingsPanel(BaseSettingsPanel):
     name = "name_email"
-    title = gettext_lazy("Name and Email")
+    # title = gettext_lazy("Name and Email")
```

Comment:
Issue: Looks like you have some unrelated changes in this PR

Reply:
These are related to [#10937](https://github.com/wagtail/wagtail/issues/10937) this issue

Reply:
Then it should be included in a separate PR, not combined in this one.

Reply:
@dhruvpal05 this PR should not have any of these changes to the account file, please can you either revert this commit or reset this file to it's original state and push.